ViennaTools/ViennaLS
# Update Documentation & Github Actions

## Features

- Use Matrix in Workflows  
  > * Reduces code duplication
  > * Test both Release & Debug configurations 
  > * Test compilation with VTK installed as system dependency

- Use CPM Cache  
  > * @actions/cache is also used instead of custom solution
  > * Also cache more aggressively (More aggressive caching achieved 3 Minute compile on Windows)

- Remove Steps that are no longer required once #99 is merged

- Update OpenMP on MacOS _(Generously taken from https://github.com/alpaka-group/alpaka)_

- Updated Readme (see https://github.com/Curve/ViennaLS/blob/actions-refactor/README.md)

- Updated Doxygen Setup (Preview: https://curve.github.io/ViennaLS/index.html)
  > Also removed the html output from the source dir

## Missing

- Invocation of Remote Workflow in ViennaPS
